ccd-config-generator/build.gradle

```diff
@@ -1,30 +1,35 @@
 
 repositories {
+    jcenter()
```

Comment:
do we need both jcenter and mavenCentral? IIRC jcenter is going away

Reply:
for now - it's needed for the ccd-client